gold-phone-input.html

```diff
@@ -236,7 +236,8 @@
 
     beforeRegister: function() {
       var template = Polymer.DomModule.import('gold-phone-input', 'template');
-      var version = Polymer.Element ? 'v1' : 'v0';
+      var ironInput = document.createElement('iron-input');
+      var version = typeof ironInput._initSlottedInput == 'function' ? 'v1' : 'v0';
```

Comment:
Is it the same problem as in `paper-input`, in the sense that we don't know which version of `iron-input` we're dealing with? if so, could you add these comments here as well? https://github.com/PolymerElements/paper-input/blob/master/paper-input.html#L257-L262